etrago/appl.py

```diff
@@ -124,6 +124,7 @@
         "cluster_foreign_AC": False,  # take foreign AC buses into account, True or False
         "method_gas": "kmedoids-dijkstra",  # choose clustering method: kmeans or kmedoids-dijkstra
         "n_clusters_gas": 14,  # total number of resulting CH4 nodes (DE+foreign)
```

Comment:
I always need at least 15 gas and H2 buses. Is that correct? If yes, I would change the default value here

Reply:
Changed in 2194e60ba83855ce305a0d886e27e39f62242913